Changes

```diff
@@ -373,6 +373,10 @@ Working version
   (Olivier Nicole, review by Miod Vallat, Sebastien Hinderer, Fabrice Buoro,
    Gabriel Scherer and KC Sivaramakrishnan)
 
+- #12819: Clarify which runtime interactions are allowed in custom ops
+  (Basile Clément, review by Guillaume Munch-Maccagnoni, Gabriel Scherer and
```

Comment:
I did not review this PR ("on behalf of" secretly means "I didn't read this in details").

Reply:
But now you did :thinking: 

I added the people involved in the discussion without giving it any more thought — removed your name.